go/libraries/doltcore/sqle/enginetest/dolt_queries.go

```diff
@@ -1183,7 +1183,7 @@ var MergeScripts = []queries.ScriptTest{
 		},
 		Assertions: []queries.ScriptTestAssertion{
 			{
-				Query:    "CALL DOLT_MERGE('feature-branch', '-m', 'this is a merge')",
+				Query:    "CALL DOLT_MERGE('feature-branch', '-m', 'this is a merge', '--no-commit')",
```

Comment:
It would be good to have a little more coverage of dolt_merge in the enginetests. I'd cover the happy path with merge creating a commit and also fast-forward. Then I'd cover the error cases where a commit couldn't be created.